spanner/api/Spanner.Samples.Tests/AlphabeticalTestOrderer.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright 2021 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+
+namespace Spanner.Samples.Tests
+{
+    public class AlphabeticalTestOrderer : ITestCaseOrderer
```

Comment:
I think you don't need this class anymore.

Reply:
Nope, removed.